feature-libs/product-configurator/rulebased/components/overview-sidebar/configurator-overview-sidebar.module.ts

```diff
@@ -0,0 +1,36 @@
+/*
+ * SPDX-FileCopyrightText: 2022 SAP Spartacus team <spartacus-team@sap.com>
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { CommonModule } from '@angular/common';
+import { NgModule } from '@angular/core';
+import { FormsModule, ReactiveFormsModule } from '@angular/forms';
+import { CmsConfig, I18nModule, provideDefaultConfig } from '@spartacus/core';
+import { ConfiguratorOverviewFilterModule } from '../overview-filter/configurator-overview-filter.module';
+import { ConfiguratorOverviewMenuModule } from '../overview-menu/configurator-overview-menu.module';
+import { ConfiguratorOverviewSidebarComponent } from './configurator-overview-sidebar.component';
+
+@NgModule({
+  imports: [
+    CommonModule,
+    I18nModule,
+    FormsModule,
+    ReactiveFormsModule,
```

Comment:
@AlexanderKuschner: do we use those 2 imports somewhere?, or a can they be removed?